src/Home.jsx

```diff
@@ -2,40 +2,76 @@ import React, { useEffect, useState } from "react";
 import axios from "axios";
 
 import ForumCard from "./ForumCard";
+import EventLink from "./EventLink";
 
 import List from "@material-ui/core/List";
 import ListItem from "@material-ui/core/ListItem";
 import useStyles from "./styles/HomeStyles";
 
 export default function Home(props) {
   const classes = useStyles();
-  const [posts, setPosts] = useState([]);
+  const [posts, setPosts] = useState([]); 
+  const [events, setEvents] = useState([]);
 
   const goToForum = (id) => {
     props.history.push(`/game/${id}`);
-  };
+  }; 
+
+  const goToEvent = (event_id) => {
+    window.location.reload();
+  }
 
   useEffect(() => {
     // Fetching available forums
     const fetchPosts = async () => {
-      const result = await axios("https://jsonplaceholder.typicode.com/posts");
+      const result = await axios("https://jsonplaceholder.typicode.com/posts"); 
       setPosts(result.data);
-    };
+    }; 
+    
+    // Fetching "events", would consist of sports: golf/soccer etc, and political events: 2020 US presidential election etc. 
+    // Following Betfair's logic
+    const fetchEvents = async () => { 
+      const result_events = await axios("https://jsonplaceholder.typicode.com/albums");  
+      result_events.data=result_events.data.slice(0, 20);
+      setEvents(result_events.data);
+    }
     console.log("FETCHING FORUM DATA");
-    fetchPosts();
+    fetchPosts(); 
+    console.log("FETCHING EVENTS DATA"); 
+    fetchEvents(); 
+    
   }, []);
 
   return (
-    <div className={classes.root}>
+    <div className={classes.root}> 
```

Comment:
We could probably consider making components for EventThreads and Hot Posts, or some generic version of them that could be reused on other pages. Fine for now, but as this file gets longer it could help keep it tidy.